Jenkinsfile

```diff
@@ -21,7 +21,7 @@ def PIPELINE_CONTROL = [
     build: true,
     unit_test: true,    
     system_test: true,
-    deploy: true,
+    cd_skip: false,
     ci_skip: false ]
```

Comment:
Suggest removing all the stuff relating `ci_skip` as we don't need this

Reply:
done